VirtusLab-Open-Source/strapi-plugin-comments
# feat: [CU-2ep5xe1] [CU-2eap4ev] inform commenter about responses & avatar fix

## Ticket

https://github.com/VirtusLab/strapi-plugin-comments/issues/142
https://github.com/VirtusLab/strapi-plugin-comments/issues/139

## Summary

What does this PR do/solve?

Introduces functionality of informing thread author about responses & fixes moderation panel crashing when `avatar` field of Strapi User is not `media` or `relation`.

## Test Plan
1. Run the app
2. Add field `avatar` with type `string` to the `User` collection and create sample users
3. Post a comment and couple responses to it
4. See that moderation panel works (displays avatars) and you're receiving e-mails about responses (depends on your SMTP configuration)
